AccessibilitySnapshot/iOSSnapshotTestCase/Classes/FBSnapshotTestCase+Accessibility.swift

```diff
@@ -53,6 +55,22 @@ extension FBSnapshotTestCase {
             return
         }
 
+        if forceCursorVisible {
+            // Show the cursor of all text fields.
+            view.recursiveForEach(viewType: UITextField.self) { textField in
+                textField.becomeFirstResponder()
+
+                // Find the field editor of the text field which controls the animation of the cursor
+                guard let fieldEditorLayer = (textField.value(forKey: "fieldEditor") as? UIView)?.layer else {
+                    return
+                }
+
+                // Set the speed and time offset of the layer such that the cursor will remain visible
+                fieldEditorLayer.speed = 0
+                fieldEditorLayer.timeOffset = fieldEditorLayer.beginTime
```

Comment:
Let's make sure to restore the original values here to avoid potentially breaking any other tests associated with the view. Maybe build up a set of `(layer: CALayer, speed: Float, timeOffset: CFTimeInterval)` here and then go through  and restore the values after the snapshot?

I'm not sure the best way to pause/resume this layer animation is. We might want to always set `timeOffset` to 0 here, and then set `beginTime` to the original `beginTime + timeOffset` when we restore it? Might need to do a bit of research to see exactly how that works.

Reply:
Added a mechanism to resume the animation after the snapshot completes